### PR TITLE
Fix example error in memoization post

### PR DESCRIPTION
**Pull request by adarsh**
Reason for Change
=================
* Friend-of-the-blog @gabebw astutely [pointed out how my irb example was not correct][1].
* In fact, the point I was making about adding to the public API is just incorrect.

[1]: https://github.com/adarsh/blog/pull/60#pullrequestreview-4155021

Changes
=======
* Remove that point for something gentler.

### CODE REVIEW

**Comment by gabebw** [comment]
> Friend-of-the-blog

That one's going in the twitter bio.

**Comment by gabebw** [comment]
👍  One additional point you may want to make, which ties in, is to use an underscore when memoizing _in Rails controllers_. Since all controller ivars are available in the view, It's often helpful to mark ivars that should only be used in the controller with a leading underscore. That also allows things like this, where `@users` is exposed to the view and `@_users` is just used for memoization:

```ruby
def index
  @posts = posts.where(user: users.where(has_post: true))
  @users = users.where(confirmed: true)
end

private

def users
  @_users = User.all
end
```

**Comment by adarsh** [comment]
@gabebw This is a marvelous point, and well-made.

I posted your comment verbatim into the article and cited you (b1a2e8f). Is this okay by you?

File: source/articles/2016/use-an-underscore-when-memoizing-in-ruby.html.md [hunk 1]
```diff
@@ -88,12 +88,42 @@ end
...
+> end
+>
+> private
+>
+> def users
+>   @_users = User.all
+> end
+> ```
+
+It's good to have smart friends.
```
Comment (gabebw) [inline source/articles/2016/use-an-underscore-when-memoizing-in-ruby.html.md]: 😊

**Comment by gabebw** [comment]
Yes, of course! Looks great!

**Pull request closed without merging**